proof/refine/ARM/Finalise_R.thy

```diff
@@ -1389,7 +1390,21 @@ lemma deletedIRQHandler_ct_not_inQ[wp]:
   apply (simp add: comp_def)
   done
 
-crunch ct_not_inQ[wp]: emptySlot "ct_not_inQ"
```

Comment:
I have no idea why this `crunch` stopped working... Any ideas on why would be interesting. I'm not sure this is used anywhere, but it might still be nice to have, considering that `ct_not_inQ` does still appear in a few places.

Reply:
What does it get stuck on? My only guess is that some other lemma that this crunch depended on now either no longer exists or has stopped being generated by a different crunch.